### DISCUSSION TITLE
Official Style Dictionary plugin

### DISCUSSION THREAD

**Discussion by kxseilerjo** [nrwl/nx#23491] [Feature Requests]
I wanted to ask if there are any plans to introduce an official plugin for Style Dictionary, or if there is interest for that in the community.

There is a [community plugin](https://github.com/nxkit/nxkit), but it's rather basic and seems to be only sporadically maintained.

Specifically what I'm missing is the option to (simply) make publishable Style Dictionaries. 

Tips and workarounds also appreciated 🙏

**Comment by astrodomas** [comment]
Agree, we're using that plugin. Would be nice if style-dictionary would be officially supported.
